utils/mock-data-loader/re-insert-mocks.js

```diff
@@ -29,8 +28,12 @@ const init = async () => {
     roles: ['data-admin'],
     email: 're-insert-mocks-data-loader-tfm@ukexportfinance.gov.uk',
   });
+
+  await cleanAllTablesTfm(tfmToken);
```

Comment:
When I looked nothing in here actually touches TFM (it goes to central) so doesn't need the token, has that changed? Or are we just future proofing ourselves here?

Reply:
refactored it for those changes